cellprofiler/modules/measureobjectintensitydistribution.py

```diff
@@ -82,10 +58,31 @@ def __image_resource(filename):
 .. |MeasureObjectIntensityDistribution_image0| image:: {MeasureObjectIntensityDistribution_Magnitude_Phase}
 .. |MeasureObjectIntensityDistribution_image1| image:: {MeasureObjectIntensityDistribution_Edges_Centers}
 
-""".format(**{
-                "MeasureObjectIntensityDistribution_Magnitude_Phase":MeasureObjectIntensityDistribution_Magnitude_Phase,
-                "MeasureObjectIntensityDistribution_Edges_Centers":MeasureObjectIntensityDistribution_Edges_Centers
-})
+"""
+
+import os
+import pkg_resources
+
+
+def __image_resource(filename):
```

Comment:
You can use `__image_resource` defined in [`cellprofiler.modules._help`](https://github.com/CellProfiler/CellProfiler/blob/master/cellprofiler/modules/_help.py#L228-L241).

Reply:
This was leftover code I just moved around, I can go ahead and import this.